handbook/openid-connect/openid-connect.md

```diff
@@ -316,3 +316,17 @@ The following sources can be used to troubleshoot a failed connection.
     SystemLink uses log rotation therefore the latest logs may be in one of the numbered `error.log` files.
 
 **Returned Claims:** See [**Viewing Claims Returned by a Provider**](#viewing-claims-returned-by-a-provider).
+
+### Cache Entry Size
+
+The OpenID Connect module stores information in a shared memory cache. If a cache entry is too large, users will see an "Internal Server Error" when attempting to log in, and the NI Web Server logs will contain the following entries
+```
+    oidc_cache_shm_set: could not store value since value size is too large
+    oidc_cache_set: could NOT store X bytes in shm cache backend for key Y
+```
+To resolve this issue
+
+- Open the following file in a text editor run as Administrator `C:\Program Files\National Instruments\Shared\Web Server\conf\defines.d\50_mod_auth_openidc-defines.conf`
+- Find the line `Define AUTH_OIDC_CACHE_ENTRY_SIZE 66065`
+- Modify `66065` to a number larger than X, where X is the required size of the cache entry specified in the error log
```

Comment:
Instead of specifying the exact number, should be say to "find the line starting with ..." and to change the number at the end?